engine/previews/citizens_advice_components/targeted_content_preview/adviser.html.haml

```diff
@@ -0,0 +1,44 @@
+= render(CitizensAdviceComponents::TargetedContent.new(type: :adviser,
+  title: "Students or self-sufficient people",
+  id: "targeted-content-adviser")) do
+  %p Students and people who can support themselves without working (either with their own money, or with help from family or friends) can have a right to reside.
+  %p One condition of this type of right to reside is that they won’t be an ‘unreasonable burden on the social assistance system’ - either they’re unlikely to claim benefits or, if they do, they won’t need them for long.
+
+  -# @TODO: Replace with table component once available
```

Comment:
Ended up inlining the table example to avoid this being blocked on the table component but we should come back and replace this with a call to the component when that's been migrated.